src/mame/audio/vicdual.cpp

```diff
@@ -554,18 +554,42 @@ borderline_audio_device::borderline_audio_device(const machine_config &mconfig,
 
 void borderline_audio_device::device_add_mconfig(machine_config &config)
 {
+#if 0
 	NETLIST_SOUND(config, "sound_nl", 48000)
+#else
+	NETLIST_SOUND(config, "sound_nl", 22050)
+#endif
```

Comment:
Did you find that this made a performance difference? In general, the output rate of the audio doesn't impact the netlist execution performance, so I would leave it at 48000 unless you are showing some notable improvement.

Reply:
sorry, it was temporary. i thought it gives me a bit more performance with it, but it doesn't... did i forgot to remove it?